frontend/cypress/tests/codeView/codeView_codeHighlighting.cy.js

```diff
@@ -0,0 +1,16 @@
+describe('code highlighting works properly', () => {
+  it('line numbers should all have the same colour', () => {
+    cy.get('.icon-button.fa-code')
+      .should('exist')
+      .first()
+      .click();
+
+    cy.get('#tab-authorship .files', { timeout: 90000 })
+      .should('be.visible');
+
+    cy.get('.line-number')
+      .each((el) => {
+        expect(el.css('color')).to.equal('rgb(158, 158, 158)'); // #9E9E9E
```

Comment:
Since the issue is regarding consistency, shall we instead ensure that all the line numbers are the same color? This way we could avoid hard coding of the colors